contents/blog/save-engineers-time.md

```diff
@@ -0,0 +1,77 @@
+---
+date: 2022-09-01
+title: "How PostHog saves an engineer’s time"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/posthog-engineering-blog.png
+featuredImageType: full
+author: ["ian-vanagas"]
+categories: ["Engineering", "Product analytics"]
+---
+
+Engineers' time is valuable. Making sure it is effectively used should be a goal for both engineers and the companies they are a part of. Product Analytics can and should be helping you save time.
+
+On top of helping understand and create better products, PostHog provides the tools to maximize and save engineers time. We do this by:
+
+1.  Empowering non-engineers to access and utilize Product Analytics themselves.
+
+2.  Making it easier for engineers to implement and customize Product Analytics.
+
+To provide details about both, here are four specific ways PostHog saves engineers' time.
+
+## Test features faster
+
+The goal of implementing a new feature is for it to improve the product. Understanding when a feature has succeeded is key to doing this. You waste less time on low-impact features. Many analytics providers make testing difficult to implement or slow to get results. PostHog's testing tools help both engineers and non-engineers test and understand what features are working fast.
+
+[Feature Flags](https://posthog.com/product/feature-flags) are an example of this. Feature Flags help test new features to a subset of users, and they integrate with the rest of your product data. PostHog provides tools to non-engineers to understand Feature Flags, help set them up, and monitor their success. They have more ability to decide the "when, where, and who" of Feature Flags. For you, all you need to add is a single [if statement](https://posthog.com/manual/feature-flags#implementing-the-feature-flag) around the feature. This leaves you more time to implement the feature itself.
```

Comment:
```suggestion
[Feature flags](https://posthog.com/product/feature-flags) are an example of this and help you test new features with a subset of users, while integrating with the rest of your product data. PostHog provides tools to help non-engineers understand feature flags, set them up, and monitor their success. They have more ability to decide the "when, where, and who" of feature flags -- all you need to add is a single [if statement](https://posthog.com/manual/feature-flags#implementing-the-feature-flag) around the feature. This leaves engineers more time to implement the feature itself.
```
I'll stop with the casing edits after this in the interest of giving higher-level edits, but we'd need to correct this throughout the article. This is actually mine and Cory's fault, as we haven't updated the style guide to reflect the new decision (see below).

<img width="424" alt="Screenshot 2022-09-02 at 10 59 00" src="https://user-images.githubusercontent.com/84011561/188115289-38f44c92-139e-4235-b428-539e13a74698.png">

Reply:
PR to fix style guide: https://github.com/PostHog/posthog.com/pull/4124